config.cc

```diff
@@ -6,6 +6,7 @@
 #include <QFile>
 #include <QtXml>
 #include "gddebug.hh"
+#include <QStandardPaths>
```

Comment:
[QStandardPaths](https://doc.qt.io/qt-5/qstandardpaths.html) was introduced in Qt 5.0. Goldendict supports building with Qt 4.6 or later.